landoapi/api/revisions.py

```diff
@@ -7,30 +7,24 @@
 """
 from connexion import problem
 from landoapi.phabricator_client import PhabricatorClient
-from landoapi.transplant_client import TransplantClient
 
 
-def get(revision_id, api_key=None):
-    """ API endpoint at /revisions/{id} to get revision data. """
+def get_revision(revision_id, api_key=None):
+    """ Gets revision from Phabricator.
+
+    Returns None or revision.
+    """
     phab = PhabricatorClient(api_key)
     revision = phab.get_revision(id=revision_id)
-
     if not revision:
-        # We could not find a matching revision.
-        return problem(
-            404,
-            'Revision not found',
-            'The requested revision does not exist',
-            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
-        )
+        return None
 
-    return _format_revision(phab, revision, include_parents=True), 200
+    return _format_revision(phab, revision, include_parents=True)
```

Comment:
If you do make a helper method for `transplants.py` remember that setting include_parents=True is very expensive because it will make several network requests. In fact, if you only need to get the revision id, then you won't need to call_format_revision at all.